src/caf/toolkit/translation.py

```diff
@@ -575,27 +575,24 @@ def pandas_long_matrix_zone_translation(
     """
     # pylint: disable=too-many-arguments, too-many-locals
     # Init
+    if bool(index_col_2_out_name) != bool(index_col_1_out_name):
```

Comment:
I don't think this causes any problems here but its worth noting some edge cases with None and an empty string both being False `bool(None) == bool("") == False`. Also, if integers are given that may also confuse things as `bool(0) == False` but any other number is True.

I think an explicit None check is usually better but in this case it would be quite verbose and I think the edge cases above are unlikely.